sdk/eventhub/testhub/commands/sendReceive.ts

```diff
@@ -3,7 +3,7 @@
 import { CommandBuilder } from "yargs";
 import { EventHubClient, EventPosition, EventData, delay } from "@azure/event-hubs";
 import { log, setCurrentCommand, randomNumberFromInterval } from "../utils/util";
-import * as uuid from "uuid/v4";
+import uuid from "uuid/v4";
```

Comment:
Why was this change needed?

Reply:
esModuleInterop getting enabled, most likely.

Reply:
It wasn't building for me at all with rollup without this change. I think the older version of rollup was okay with it, but the newer version uses esModules everywhere internally